sda-commons-starter-web/src/main/resources/org/sdase/commons/spring/boot/web/monitoring/monitoring.properties

```diff
@@ -3,9 +3,9 @@ management.server.port=8081
 management.server.base-path=/
 management.endpoints.web.base-path=/
 management.endpoints.web.exposure.include=*
-management.endpoints.enabled-by-default=false
+management.endpoints.access.default=none
 # Healthcheck
```

Comment:
In Spring Boot 3.4 Release Notes it's mention that these properties is deprecated so I made this change as mentioned [here](https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-3.4-Release-Notes#:~:text=Controlling%20Access%20to,endpoint.%3Cid%3E.access)